### PR TITLE
[#205] Set Rectangle Color (Diff / Exclude)

### PR DESCRIPTION
**Pull request by woosyume**
# PR Details

## Description
As a new feature, I'm suggesting that client can set color for rectangles with the difference.

## Related Issue
https://github.com/romankh3/image-comparison/issues/205

## Motivation and Context
About my case,
When we release something, we run image comparison for before and after deployment.
It's job of internal automation app based on Selenium WebDriver.

But we sometime take time to find diff rectangle when it's exists because there are some other parts with red colored.
Maybe you can see it with the following page.
https://www.rakuten.co.jp/

That's why I want to set other color so that we can find difference as soon as possible.

## How Has This Been Tested
Tested in UT level

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
Screen Shot 2021-03-25 at 17 48 23

### CODE REVIEW

**Review by romankh3 — APPROVED** [review]
That's good

File: src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java [hunk 1]
```diff
@@ -464,7 +465,9 @@ public void shouldProperlyWorkGettersAndSetters() {
                 .setThreshold(400)
                 .setDifferenceRectangleFilling(true, 35.1)
                 .setExcludedRectangleFilling(true, 45.1)
-                .setAllowingPercentOfDifferentPixels(48.15);
+                .setAllowingPercentOfDifferentPixels(48.15)
+                .setDifferenceRectangleColor(Color.RED)
```
Comment (romankh3) [inline src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java:469]: It's good to set a non-default value here. The same for line 469.
Please, update it.

**Review by romankh3 — COMMENTED** [review]
Please. add a test with pictures for `ImageComparison.compareImages()` method with modified colors for rectangles.

File: src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java (hunk 1 above)
Comment (woosyume) [reply src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java:469]: Thank you for your comment.
Could you please let me know that you mean only set a non-default value for test class or for main modification as well?

Comment (romankh3) [reply src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java:469]: Only for test case to be ensure, that setter works as well as we expected. 

Default values for production code should be the same as were before.

**Pull request merged**